### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#80] [Source Q&A] [answered]
@DEMOLITIONDON96 
Would you mind looking a bit in your freeplay category ? I won't use any art, i would just like to add it in mt engine

**Comment by ThatOneSillyGuyDon** [comment, marked as answer]
It’s not even done, and very unstable, so, I wouldn’t add it yet

**Reply by Theoyeah** [reply]
ok, but when it will be finished, could i do it ?

**Reply by ThatOneSillyGuyDon** [reply]
Sure, after all, FNF is open source, so anyone can take anything as long as it’s credited

**Reply by Theoyeah** [reply]
thanks

**Reply by ghost** [reply]
> Sure, after all, FNF is open source, so anyone can take anything as long as it’s credited

Because of the Apache 2.0 licence.
